swap/src/cli/command.rs

```diff
@@ -109,18 +106,10 @@ pub enum Command {
 
 #[derive(structopt::StructOpt, Debug)]
 pub struct AliceConnectParams {
-    #[structopt(
-        long = "seller-peer-id",
-        default_value = DEFAULT_ALICE_PEER_ID,
-        help = "The peer id of a specific swap partner can be optionally provided"
-    )]
+    #[structopt(long = "seller-peer-id", help = "The sellers peer identity")]
```

Comment:
```suggestion
    #[structopt(long = "seller-peer-id", help = "The seller peer identity")]
```

Since it is always one specific seller, I'd say it should be `seller's` or `seller`.